tests/apichecks/testdata/exceptions/acronyms.txt

```diff
@@ -156,17 +164,17 @@
-[acronyms] crd=computeinstances.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].internalIpv6PrefixLength" should be ".spec.networkInterface[].internalIPV6PrefixLength"
-[acronyms] crd=computeinstances.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].ipv6AccessConfig[].externalIpv6" should be ".spec.networkInterface[].ipv6AccessConfig[].externalIPV6"
-[acronyms] crd=computeinstances.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].ipv6AccessConfig[].externalIpv6PrefixLength" should be ".spec.networkInterface[].ipv6AccessConfig[].externalIPV6PrefixLength"
+[acronyms] crd=computeinstances.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].internalIpv6PrefixLength" should be ".spec.networkInterface[].internalIPv6PrefixLength"
+[acronyms] crd=computeinstances.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].ipv6AccessConfig[].externalIpv6" should be ".spec.networkInterface[].ipv6AccessConfig[].externalIPv6"
+[acronyms] crd=computeinstances.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].ipv6AccessConfig[].externalIpv6PrefixLength" should be ".spec.networkInterface[].ipv6AccessConfig[].externalIPv6PrefixLength"
 [acronyms] crd=computeinstances.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].networkIp" should be ".spec.networkInterface[].networkIP"
 [acronyms] crd=computeinstances.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].networkIpRef" should be ".spec.networkInterface[].networkIPRef"
 [acronyms] crd=computeinstances.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].networkIpRef.external" should be ".spec.networkInterface[].networkIPRef.external"
 [acronyms] crd=computeinstances.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].networkIpRef.kind" should be ".spec.networkInterface[].networkIPRef.kind"
 [acronyms] crd=computeinstances.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].networkIpRef.name" should be ".spec.networkInterface[].networkIPRef.name"
 [acronyms] crd=computeinstances.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].networkIpRef.namespace" should be ".spec.networkInterface[].networkIPRef.namespace"
 [acronyms] crd=computeinstances.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.scheduling.minNodeCpus" should be ".spec.scheduling.minNodeCPUs"
-[acronyms] crd=computeinstances.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.shieldedInstanceConfig.enableVtpm" should be ".spec.shieldedInstanceConfig.enableVTPM"
+[acronyms] crd=computeinstances.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.shieldedInstanceConfig.enableVtpm" should be ".spec.shieldedInstanceConfig.enablevTPM"
```

Comment:
So this one actually looks wrong to me.  It's an odd case though:  The acronym is "Virtual Trusted Platform Module", and I think the lower case v is just a "that looks cool" thing.

Similarly, I think it is "Internal Protocol Version 6", I don't quite know how to parse IPv6 - is it "I" "Pv" "6" ?

I wonder whether k/k has had to tackle this on any fields?